pkg/defaultmodule/module.go

```diff
@@ -95,6 +95,11 @@ func (*Module) CloudEventConvert(_ context.Context, msgData []byte) ([]cloudeven
 		statusHdr.Type = cloudevent.TypeStatus
 		hdrs = append(hdrs, statusHdr)
 	}
+	if gjson.GetBytes(event.Data, "eventCategory").Exists() {
```

Comment:
`eventCategory` is a bit confusing for a name. I am not sure what this means. I would be fine with something as simple like `events`